scripts/cromwell/analyze_resource_acquisition.py

```diff
@@ -0,0 +1,460 @@
+#!/bin/python
+
+import json
+import argparse
+import numpy as np
+import dateutil.parser
+from dateutil.tz import tzutc
+import datetime
+import time
+import pandas as pd
+import math
+import random
+import matplotlib.pyplot as plt
+import matplotlib.ticker
+import sys
+from os.path import basename, isfile, getsize
+SEED = 42
```

Comment:
…I don't know, actually. I think it might have been relevant in a very early draft, or copied from another script. It doesn't seem to change anything now, so I removed it.